independent-publisher-connectors/Mensagia/apiProperties.json

```diff
@@ -1,25 +1,25 @@
 {
-  "properties": {
-    "connectionParameters": {
-      "api_key": {
-        "type": "securestring",
-        "uiDefinition": {
-          "displayName": "Api Key",
-          "description": "The API Key for this api",
-          "tooltip": "Provide your API Key",
-          "constraints": {
-            "tabIndex": 2,
-            "clearText": false,
-            "required": "true"
-          }
-        }
-      }
-    },
-    "iconBrandColor": "#da3b01",
-    "capabilities": [
-      "actions"
-    ],
-    "publisher": "Sistemas Informaticos ICON, S.L.",
-    "stackOwner": "Sinergyne Global Communications, S.L."
-  }
-}
+    "properties": {
```

Comment:
@Sistemes-Icon Indentation - The formatting of both the swagger and apiProperties.json files is not correct. The properties are not aligned properly and there is a longer tab index between the properties. Please refer to the certified connectors in the [open source repo](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/certified-connectors/Africa%27s%20Talking%20Airtime/apiDefinition.swagger.json) for reference.